### PR TITLE
Create Greedy Algorithm

### PR DESCRIPTION
**Pull request by shahbazalam07**
Added Greedy Algorithm problem(Coin Change Problem) by Java.

### CODE REVIEW

File: Task3/Greedy Algorithm [hunk 1]
```diff
@@ -0,0 +1,23 @@
+class Coin {
```
Comment (chandrikadeb7) [inline Task3/Greedy Algorithm:1]: Please give correct extension to code file (.java in this case)

Comment (shahbazalam07) [reply Task3/Greedy Algorithm:1]: Ok.

**Comment by shahbazalam07** [comment]
please merge my PR

File: Task3/Greedy Algorithm (hunk 1 above)
Comment (chandrikadeb7) [inline Task3/Greedy Algorithm:1]: Please follow folder structure and add the file extension .java

**Review by nimishbongale — APPROVED** [review]
Good work @shahbazalam07! 🚀 

Since it was a simple change, I've done it from my end. However, be sure to properly read the `README.md` or `CONTRIBUTING.md` file before raising the PR!

Congrats, you are now all set to contribute to major Open Source projects!

**Pull request merged**